addons/web/static/tests/views/calendar/calendar_quick_create_tests.js

```diff
@@ -116,4 +116,16 @@ QUnit.module("CalendarView - QuickCreate", ({ beforeEach }) => {
         await click(target, ".o-calendar-quick-create--cancel-btn");
         assert.verifySteps(["close"]);
     });
+
+    QUnit.test("check default title", async (assert) => {
```

Comment:
Would be better to test a calendar view with the special context and check that the quick create receives the correct title.